docs/javascript/generated/ComPriFacNotConcept.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * :en: Class ComPriFacNotConcept: Underlying concept of ComPriFacNot.
+ * :de: Klasse ComPriFacNotConcept: Zugrundeliegendes ComPriFacNot-Konzept.
+ *
+ * @author See git history
+ * @version 1.0, 2021-12-11
+ * @since 1.0, 2021-12-10
+ */
+class ComPriFacNotConcept {
+    /**
+     * en: Constructor.
+     * de: Konstruktor.
+     */
+    constructor() { }
+    /**
+     * :en: Initializes this class and calls a call-back-function afterwards.
+     * :de: Initialisiert diese Klasse und ruft anschließend eine Call-Back-Funktion auf.
+     *
+     * @param {any} pvCallBack Call-back-function
+     */
+    static initialize(pvCallBack) {
+        const lcRequest = new XMLHttpRequest();

```

Comment:
Unfortunately the only way to read a external resource into the runtime environment is by the means of AJAX, so that complicated way with asynch and call-back-function. The basic digits were outsourced to an external file (`basic-digits.jsonc`) to enable the use of this file by different programming languages. Unfortunately HTTP/HTTPS are the only supported protocols. The FILE protocol is protected for security reasons, so the JavaScript solution works only with a local HTTP daemon like "Live server" (an extension for Visual Studio Code).

Reply:
![image](https://user-images.githubusercontent.com/95105802/145711406-4a75735c-77a5-40de-923a-ce275786d973.png)